examples/make_generic_contacts.py

```diff
@@ -93,28 +97,73 @@ def make_contacts_by_social_layers_and_age_mixing(n=1e4,location='seattle_metro'
     modeling framework. 
 
     """
-    popdict = sp.make_popdict(n=n,location=location,state_location=state_location,country_location=country_location,use_demography=True)
+    popdict = sp.make_popdict(n=n, location=location, state_location=state_location, country_location=country_location,
+                              use_demography=True)
 
-    n_contacts_dic = {'H': 7, 'S': 20, 'W': 20, 'C': 10} # dict of the average number of contacts per layer
+    n_contacts_dic = {'H': 7, 'S': 20, 'W': 20, 'C': 10}  # dict of the average number of contacts per layer
     options_args = {'use_age_mixing': True, 'use_social_layers': True}
 
-    contacts = sp.make_contacts(popdict,n_contacts_dic=n_contacts_dic,location=location,state_location=state_location,country_location=country_location,sheet_name=sheet_name,options_args=options_args)
-    
+    popdict = sp.make_contacts(popdict, n_contacts_dic=n_contacts_dic, location=location,
+                               state_location=state_location, country_location=country_location, sheet_name=sheet_name,
+                               options_args=options_args)
+
     if verbose:
-        layers = ['H','S','W','C']
-        uids = contacts.keys()
+        layers = ['H', 'S', 'W', 'C']
+        uids = popdict.keys()
         uids = [uid for uid in uids]
         for uid in uids:
             print(uid)
             for k in layers:
-                print(k,contacts[uid]['contacts'][k])
+                print(k, popdict[uid]['contacts'][k])
+
+    return popdict
 
-    return contacts
 
+def describe_population(popdict):
+    ages = []
+    males = 0
+    locations = {}
+    contact_types = {}
+    for uid in popdict:
+        person = popdict[uid]
+        ages.append(person['age'])
+        if person['sex'] == 1:
+            males += 1
+            pass
+        location = person['loc']
+        if location:
+            if location in locations:
+                locations[location] += 1
+            else:
+                locations[location] = 1
+                pass
+            pass
```

Comment:
ditto (understand the rationale for using pass for longer loops, but here does not seem needed)